docs/user/dql/functions.rst

```diff
@@ -2436,6 +2436,28 @@ Example::
     | 2020-08-26 13:49:00                | 2020-08-27 02:04:42                                  |
     +------------------------------------+------------------------------------------------------+
 
+TIMESTAMPADD
+------------
+
+
+Description
+>>>>>>>>>>>
+
+Usage: Returns a DATETIME value based on a passed in DATE/DATETIME/TIME/TIMESTAMP/STRING argument and an INTERVAL and INTEGER argument which determine the amount of time to be added.
+If an third argument of type STRING is provided, it must be formatted as a valid DATETIME. If only a TIME is provided, a DATETIME is still returned with the DATE portion filled in using the current date.
```

Comment:
Please, reword.
I read this:
> If third argument is given

Should be
> If third argument is a STRING

DATE is converted to DATETIME automatically too (mention/refer how).